index.html

```diff
@@ -83,95 +69,172 @@
-      options.yearRange = d3.event.target.value;
-      update();
+    // format x axis
+    var formatyears = d3.format("1000")
+    // format y axis
+    var formatmoney = d3.format("$1,000,000,000")
+    //Define X axis
+   
+    var xAxis = d3.svg.axis()
+             .scale(chart.x)
+             .orient("bottom")
+             .ticks(20)
+             .tickSize(height + 15)
+             .tickFormat(formatyears);
+    //Define Y axis
+    var yAxis = d3.svg.axis()
+             .scale(chart.y)
+             .tickSize(width)
+             .tickFormat(formatmoney)
+             .orient("right")
+             .ticks(5);
+   
+    //Create X axis
+     gx = this.svg.append("g")
+          .attr("class", "axis")
+          .attr("transform", "translate(0," + height + ")")
+          .append("text")
+          .attr("x", width)
+          .attr("y", 0)
+          .style("text-anchor", "end")
+          .text("Year");
+    //Create y axis
+      gy = this.svg.append("g")
+      .attr("transform", "rotate(-90)")
+      .attr("class", "axis")
+      .append("text")
+      .style("text-anchor", "end")
+      .text("Incentive Value") 
+      .attr("x", -10)
+      .attr("y", -10);
+      gy = this.svg.append("g")
+      .attr("class", "y axis")
+      .call(yAxis);
+    gy.selectAll("g").filter(function(d) { return d; })
+      .classed("minor", true);
+    
+      gx = this.svg.append("g")
+      .attr("class", "x axis")
+      .call(xAxis);
+      
+    gx.select('g').filter(function(d) { return d; })
+        .classed("minor", true);
+    gy.selectAll("text")
+    .attr("x", 4)
+    .attr("dy", -4);
+      // EVENT HANDLERS
+    d3.select('#location').on('change', function () {
+      options.location = d3.event.target.value;
+
+
+      chart.tooltip = d3.tip()
+         .attr('class', 'tooltip')
+         .offset([-6, 0])
+         .html(function(d) { return d.length });
+ 
+     chart.tooltip(chart.svg);
+ 
+     chart.update();
     });
   }
-
-  function update() {
+    Chart.prototype.update = function() {
+      var chart = this;
     // TRANSFORM
     txData = data.slice();
-
-    if (options.yearRange !== 'all') {
-      var yearRange = options.yearRange.split('-');
-      txData = txData.filter(function (d) {
-        return d.year >= yearRange[0] && d.year <= yearRange[1];
+    if (options.location !== 'all') {
+       txData = txData.filter(function (d) {
+        return d.location === options.location;
       });
     }
-
     // SCATTERPLOT
-
-    var points = svg.selectAll('.point')
+    points = this.svg.selectAll('.point')
       .data(txData);
-
     points.enter().append('circle')
       .attr('class', 'point')
       .attr('r', 5);
-
     points
-      .attr('cx', function (d) { return x(d.unemployment); })
-      .attr('cy', function (d) { return y(d.inflation); });
-
+      .attr('cx', function (d) { return x(d.year); })
+      .attr('cy', function (d) { return y(d.value); });
```

Comment:
`x` and `y` were defined in the constructor function's scope, and to make them available here, we've saved them in the instance as `chart.x` and `chart.y`.